MHCollapsibleViewManager.m

```diff
@@ -212,7 +298,7 @@ -(MHTableViewCell*)returnCellWithIndex:(NSIndexPath*)indexPath tableView:(UITabl
         type = CRUCellViewInteractionHeader;
         collapsed = self.expanded;
         cellText = self.headerTitle;
-        detailText = [NSString stringWithFormat:NSLocalizedString(@"%i items",), self.filterSections.count];
```

Comment:
I know you removed it but I want to encourage you to us NSLocalizedString for all visible text in the app. We are trying to make our apps translatable. It is a good habit to get into. However, there are many ways to go about it. We use the keyed route. Where instead of using the text directly we use a key eg
```
NSLocalizedString(@"MHFilterViewController_Interaction_CellHeader_defaultText",nil);
```
Then in the .strings file you would add a line
```
MHFilterViewController_Interaction_CellHeader_defaultText = "%i items";
```
That way we can change the english without having to update every other language file.

The alternative is to use the english word as the key eg:
```
NSLocalizedString(@"Their is no problem",nil);
```
Then in the english .strings file you would add a line
```
"Their is no problem" = "Their is no problem";
```
and in the fr you would have
```
"Their is no problem" = "Leur est pas un problème";
```
But if you wanted to change the grammatical error you have to change it lots of places not just once because we used the english phrase as the key.

Even though the second method makes the code more readable we don't use it because it is time consuming to make changes (it also limits context).